user_management/api/tests/test_serializers.py

```diff
@@ -161,7 +162,8 @@ def test_deserialize_mismatched_passwords(self):
         self.data['password2'] = 'different_password'
         serializer = serializers.RegistrationSerializer(data=self.data)
         self.assertFalse(serializer.is_valid())
-        self.assertIn('password2', serializer.errors)
+        message = 'Your passwords do not match.'
+        self.assertIn(message, serializer.errors['non_field_errors'])
```

Comment:
It's a list of that message, yes.  I'm keen on allowing this test to be generic though - we don't care that there aren't _more_ errors in `non_field_errors`, only that this one is present.